apps/api/src/bootstrap.ts

```diff
@@ -1,6 +1,6 @@
+import 'newrelic';
```

Comment:
You can also move the import to `./instrument` and import `./instrument` as the first thing.

Reply:
Shall I move that inside instrument too?

Reply:
Yes